commands.js

```diff
@@ -1111,7 +1111,8 @@ var commands = exports.commands = {
 	promotehelp: ["/promote [username], [group] - Promotes the user to the specified group. Requires: & ~"],
 
 	globaldemote: 'demote',
-	demote: function () {
+	demote: function (target) {
```

Comment:
Yes is a fix, because when you do /demote without a target, its show the promote help not the demote help.